src/__tests__/ui-tests/contactUsPage.test.js

```diff
@@ -0,0 +1,44 @@
+const { assert } = require("chai");
+const fs = require("fs");
+const path = require("path");
+const chrome = require("selenium-webdriver/chrome");
+const { Builder, By } = require("selenium-webdriver");
+
+const myoptions = new chrome.Options().headless().addArguments("disable-gpu");
+const TIMEOUT = 8000;
+let driver;
+
+beforeAll(async function () {
+  driver = await new Builder()
+    .forBrowser("chrome")
+    .setChromeOptions(myoptions)
+    .build();
+  await driver
+    .manage()
+    .setTimeouts({ implicit: TIMEOUT, pageLoad: TIMEOUT, script: TIMEOUT });
+});
+
+afterAll(async function () {
+  await driver.quit();
+});
+
+describe("Selenium Tests for Contact Us Page", function () {
+  it("Should be able to navigate to the Contact Us Us Page from the Hero Section", async function () {
+    // Navigate to the home page
+    await driver.get("http://localhost:3000/");
+    await driver.manage().window().setRect({ width: 1401, height: 800 });
+    await driver.findElement(By.xpath("/html/body/div[1]/div/nav/div[2]/ul/li[5]/a")).click();
```

Comment:
Don't use X paths they rely on page structure which is subject to change, add an id for the button and find by ID